src/chrome/content/rules/xael.org.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="xael.org">
+	<target host="xael.org" />
+	<target host="www.xael.org" />
```

Comment:
`photos.xael.org`
`stats.xael.org`

Reply:
https://travis-ci.org/EFForg/https-everywhere/jobs/586367665#L320

Refused all?

Reply:
Yes, I think it may be closed until the site returns online.